BitwardenShared/UI/Vault/VaultItem/EditCollections/EditCollectionsView.swift

```diff
@@ -51,6 +51,7 @@ struct EditCollectionsView: View {
                             Text(collection.name)
                         }
                         .toggleStyle(.bitwarden)
+                        .accessibilityIdentifier("CollectionItemCell")
```

Comment:
🤔 Correct me if I'm wrong but if there are multiple collections all of them will have the same A11y identifier. Is that alright or should a number or something else be added to differentiate each collection cell?

Reply:
1. We expect each collection toggle to have the same locator regardless of where it is displayed. The `EditCollectionView` did not have the expected identifier, so the CollectionItem component is unable to select collections there.
2. You are right. The concept of `Cell` we use in our framework to refer to the collection list items was inherited from the component we made for the `Xamarin.Forms` app, and it's obsolete. I'll update the locator in the next commit. 

Thanks, Fede!